rotkehlchen/chain/ethereum/modules/yearn/decoder.py

```diff
@@ -207,6 +297,17 @@ def _decode_vault_event(self, context: DecoderContext) -> DecodingOutput:
                 events=context.decoded_events,
                 counterparty=CPT_YEARN_V3,
             )
+        elif context.tx_log.topics[0] == ERC20_OR_ERC721_TRANSFER:
+            for tx_log in context.all_logs:
+                if tx_log.topics[0] in {
+                    YEARN_V3_WITHDRAW_TOPIC,
+                    YEARN_V2_DEPOSIT_TOPIC,
+                    YEARN_V3_DEPOSIT_TOPIC,
+                    YEARN_V2_INCREASE_DEPOSIT_TOPIC,
+                }:  # Skip transfer decoding if deposit/withdraw topics are present
+                    return DEFAULT_DECODING_OUTPUT
```

Comment:
as per the discussion in discord seems that v3 doesn't need this logic so it should be removed